docs/application/native/guides/text-input/autofill.md

```diff
@@ -0,0 +1,365 @@
+# Autofill
+
+Autofill is a feature that allows you to fill out commonly-entered information such as email, account and address in a text input field. email, account and address in a text input field.
+Using autofill functions, you can get the retrieve the data previously entered by the user and stored or request to store data in an autofill service.
+
+The main features of the Autofill API include:
+
+- [Requesting to get the presence of autofill data and autofill authentication information](#requesting-to-get-the-presence-of-autofill-data-and-autofill-authentication-information)
+
+  You can get the autofill service information such as autofill service name, service message and service logo image to use `autofill_auth_info_request()`.
+  You can get the presence of autofill data for each view in autofill service information to be received.
+
+- [Retrieving the data previously entered by the user and stored in an autofill service](#retrieving-the-data-previously-entered-by-the-user-and-stored-in-an-autofill-service)
+
+  Use the `autofill_fill_request()` function to retrieve the data previously entered by the user and stored in an autofill service
+
+- [Requesting to store the data entered by the user in autofill service](#requesting-to-store-the-data-entered-by-the-user-in-autofill-service)
+
+  Use the `autofill_commit()` function to store the data entered by the user in autofill service
+
+## Prerequisites
+
+1. Configure autofill service:
+
+   Set autofill service in Settings > Language and input > Input Assistance > Autofill setting.
+   In some environment, the autofill service can be designated according to the system configuration.
```

Comment:
In some other environments, the Autofill service can be found within the system configuration